molecule/src/molecule_ionize.cpp

```diff
@@ -444,433 +419,423 @@ void MoleculePkaModel::_loadSimplePkaModel()
     _model.simple_model_ready = true;
 }
 
-void MoleculePkaModel::_loadAdvancedPkaModel()
+namespace  // only for data
```

Comment:
This namespace (by its designation) should be inner of `namespace indigo`, but that one is only referred by `using namespace indigo`.

Reply:
This data/namespace is for the local usage only.